docs/about.rst

```diff
@@ -99,6 +99,13 @@ Please adhere to the following guidelines:
   reason. As another example, do not add a test that makes dozens of concurrent
   requests to a public service such as docker hub.
 
+These next guidelines **are not** mandatory, but will match the formatting adopted
+in the existing code base.
+
+* Docstrings should not end with a blank line.
+* Line length should be line-wrapped at 79 characters.
```

Comment:
I just re-read this and realized that it sounds a bit awkward. Can it read "lines should be hard wrapped at 79 characters?" Please and thank you.

Reply:
Done.